src/DonationForms/Migrations/CleanMultipleSlashesOnDB.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Give\DonationForms\Migrations;
+
+use Give\DonationForms\Actions\SanitizeDonationFormPreviewRequest;
+use Give\DonationForms\Models\DonationForm;
+use Give\DonationForms\Properties\FormSettings;
+use Give\Framework\Blocks\BlockCollection;
+use Give\Framework\Exceptions\Primitives\Exception;
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class CleanMultipleSlashesOnDB extends Migration
+{
+
+    /**
+     * @unreleased
+     *
+     * @throws Exception
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $formMetaTable = $wpdb->prefix . 'give_formmeta';
+
+        $sql = $wpdb->prepare("SELECT form_id FROM $formMetaTable as fm WHERE
+        (fm.meta_key = 'formBuilderSettings' AND fm.meta_value LIKE '%\\\\\\\\\\\\\\\\\\\\\\\\%')
+        OR
```

Comment:
Are we confident in looking for this many slashes?

Reply:
For some reason, these 24 backslashes became 12 when we passed it to the `$sql` var, and became 6 when it runs on the database _which represents a sequence of only 3 backslashes_ as we need 2 of them to escape just 1 as you can check here: 

![image](https://github.com/impress-org/givewp/assets/16308864/9108e0e0-06c1-435c-aa01-f03cc6f89bb2)

https://dev.mysql.com/doc/refman/8.0/en/string-literals.html